Document/0x05g-Testing-Network-Communication.md

```diff
@@ -621,9 +617,10 @@ When you do not have the source code:
 
 ##### OWASP MASVS
 
-- V5.3: "The app verifies the X.509 certificate of the remote endpoint when the secure channel is established. Only certificates signed by a trusted CA are accepted."
-- V5.4: "The app either uses its own certificate store or pins the endpoint certificate or public key, and subsequently does not establish connections with endpoints that offer a different certificate or key, even if signed by a trusted CA."
-- V5.6: "The app only depends on up-to-date connectivity and security libraries."
+- MSTG-NETWORK-2: "The TLS settings are in line with current best practices, or as close as possible if the mobile operating system does not support the recommended standards."
+- MSTG-NETWORK-3: "The app verifies the X.509 certificate of the remote endpoint when the secure channel is established. Only certificates signed by a trusted CA are accepted."
+- MSTG-NETWORK-4: "The app either uses its own certificate store or pins the endpoint certificate or public key, and subsequently does not establish connections with endpoints that offer a different certificate or key, even if signed by a trusted CA."
+- MSTG-NETWORK-6: "The app only depends on up-to-date connectivity and security libraries."
```

Comment:
Good point. I changed now "### Testing the Security Provider" to MSTG-NETWORK-6 instead of MSTG-NETWORK-2. That makes more sense. 
MSTG-NETWORK-2 is already covered in 0x04f